### ISSUE TITLE
Can't seem to use this in .NET standard PCL

### ISSUE THREAD

**Issue by powerdude** [Fody/PropertyChanged#205]
Hi,

when trying to use this library in a PCL that targets .net standard 1.3, you get the following error when compiling:  
```1>    Fody: Fody (version 1.29.2.0) Executing
1>MSBUILD : error : Fody: An unhandled exception occurred:
1>MSBUILD : error : Exception:
1>MSBUILD : error : Object reference not set to an instance of an object.
1>MSBUILD : error : StackTrace:
1>MSBUILD : error :    at ModuleWeaver.FindCoreReferences() in C:\projects\propertychanged\PropertyChanged.Fody\MsCoreReferenceFinder.cs:line 0
1>MSBUILD : error :    at ModuleWeaver.Execute() in C:\projects\propertychanged\PropertyChanged.Fody\ModuleWeaver.cs:line 26
1>MSBUILD : error :    at lambda_method(Closure , Object )
1>MSBUILD : error :    at InnerWeaver.ExecuteWeavers() in c:\ConsoleBuildAgent\work\ed448661dbb30d2e\FodyIsolated\InnerWeaver.cs:line 164
1>MSBUILD : error :    at InnerWeaver.Execute() in c:\ConsoleBuildAgent\work\ed448661dbb30d2e\FodyIsolated\InnerWeaver.cs:line 82
1>MSBUILD : error : Source:
1>MSBUILD : error : PropertyChanged.Fody
1>MSBUILD : error : TargetSite:
1>MSBUILD : error : Void FindCoreReferences()
1>MSBUILD : error : 
1>    Fody:   Finished Fody 21ms.

```
```

**Comment by mmacneil** [comment]
@powerdude did you have any luck with this?  Hitting the exact same thing in a xamarin forms project.

**Comment by eladmarg** [comment]
same issue for me.

**Comment by powerdude** [comment]
@mmacneil nope

**Comment by MihaMarkic** [comment]
Note that it is either PCL or .NET Standard. Not both. :)
There is a prerelease NuGet 2.0.0. that supports .NET Standard.

**Comment by SimonCropp** [comment]
fixed in  2.0.0-beta001
